src/v/kafka/server/handlers/topics/types.cc

```diff
@@ -280,6 +290,9 @@ to_cluster_type(const creatable_topic& t) {
       = get_enum_value<model::iceberg_invalid_record_action>(
         config_entries, topic_property_iceberg_invalid_record_action);
 
+    cfg.properties.iceberg_target_lag_ms = get_iceberg_target_lag_ms(
```

Comment:
why this is not just `get_duration_value` like other fields above?

Reply:
just missed it. good catch!